.nvmrc

```diff
@@ -1 +1 @@
-16.20.0
\ No newline at end of file
+18.16.0
```

Comment:
If the node version is being upgraded to 18 here, we'll need to do the same in the [Dockerfile](https://github.com/hmcts/pcq-frontend/blob/master/Dockerfile#L2)